cli/src/cli.rs

```diff
@@ -0,0 +1,64 @@
+use clap::{crate_authors, crate_version, Parser};
+use color_eyre::{eyre::eyre, Report, Result};
+use std::path::PathBuf;
+
+use crate::config::{Config, FromFile, LoggerConfig};
+const VERSION: &str = concat!(
+    "BOB-GUI VERSION: ",
+    crate_version!(),
+    "\n",
+    "BUILT AT: ",
+    env!("BUILD_TIME"),
```

Comment:
For build time it is better to use this crate: https://crates.io/crates/build-time